README.md

```diff
@@ -2011,6 +2024,142 @@ Content-Length: 8286
 }
 ```
 
+cURL:
+
+```
+$ curl "http://localhost:8000/api/workflows/v1/b3e45584-9450-4e73-9523-fc3ccf749848/metadata?includeKey=inputs&includeKey=outputs"
+```
+
+HTTPie:
+
+```
+$ http "http://localhost:8000/api/workflows/v1/b3e45584-9450-4e73-9523-fc3ccf749848/metadata?includeKey=inputs&includeKey=outputs"
+```
+
+Response:
+```
+HTTP/1.1 200 OK
+Server spray-can/1.3.3 is not blacklisted
+Server: spray-can/1.3.3
+Date: Thu, 01 Oct 2015 22:19:07 GMT
+Content-Type: application/json; charset=UTF-8
+Content-Length: 8286
+{
```

Comment:
Could the example in the README be condensed, perhaps scattered 2 ways or something like that?